### ISSUE TITLE
[bug] bundler: deb package depends on umask value on build machine

### ISSUE THREAD

**Issue by olivierlemasle** [tauri-apps/tauri#7992]
### Describe the bug

When building a Debian package on a machine where `umask` is **not** `022`, the permissions used in the .deb are non-standard.

As reported by lintian, the **permissions on the control files** are an error. For other files, it's  only a warning (non-standard permissions); however, permissions on the target system should not depend on **`umask` on the build system**:
```
E: demo-tauri: control-file-has-bad-permissions 0664 != 0644 [md5sums]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/bin/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/applications/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/128x128/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/128x128/apps/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/256x256@2/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/256x256@2/apps/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/32x32/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/32x32/apps/]
W: demo-tauri: non-standard-dir-perm 0775 != 0755 [usr/share/icons/hicolor/]
W: demo-tauri: non-standard-executable-perm 0775 != 0755 [usr/bin/demo-tauri]
W: demo-tauri: non-standard-file-perm 0664 != 0644 [usr/share/applications/demo-tauri.desktop]
```

### Reproduction

On a Linux system:

1. Set umask different than `022`:
   ```
   $ umask 002
   ```
3. Build a Tauri application
4. Inspect the package, or run `lintian`.

### Expected behavior

Standard permissions should be used in the .deb package, regardless of `umask` value.

Probably:
- Control files: `0644`
- Data:
  - Binaries (main binary + sidecar binaries): `0755`
  - Directories: `0755`
  - Generated files (icons, .desktop file): `0644`
  - Copied files (resources and `deb > files`): copied from the source file, as it is currently

### Platform and versions

```text
[✔] Environment
    - OS: Fedora 38.0.0 X64
    ✔ webkit2gtk-4.0: 2.42.1
    ✔ rsvg2: 2.56.3
    ✔ rustc: 1.72.1 (d5c2e9c34 2023-09-13)
    ✔ Cargo: 1.72.1 (103a7ff2e 2023-08-15)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: stable-x86_64-unknown-linux-gnu (default)
    - node: 18.18.0
    - pnpm: 8.8.0
    - yarn: 1.22.19
    - npm: 9.8.1

[-] Packages
    - tauri [RUST]: 1.5.1
    - tauri-build [RUST]: 1.5.0
    - wry [RUST]: 0.24.4
    - tao [RUST]: 0.16.4
    - tauri-cli [RUST]: not installed!
    - @tauri-apps/api [NPM]: 1.5.0
    - @tauri-apps/cli [NPM]: 1.5.2

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist
    - devPath: http://localhost:1420/
    - framework: React
    - bundler: Vite
```

**Comment by naman-crabnebula** [comment]
Hi @olivierlemasle , it should be fixed now.

Please verify, so that we can close this issue.

Thank You

**Comment by olivierlemasle** [comment]
Fixed with #8585.
